docs/try-it-out/quickstart.md

```diff
@@ -0,0 +1,64 @@
+# The very quick quickstart
+
+This quickstart gives you a very quick taste of n8n. Its aim is to allow you to try out the UI, and introduce you to two key features: workflow templates and expressions. It doesn't include detailed explanations or explore concepts.
+
+You will:
+
+* Install the desktop app
+* Load a workflow from the workflow templates library
+* Add a node and configure it using expressions.
+* Run your first workflow
+
+## Step one: Install and run n8n
+
+--8<-- "_snippets/try-it-out/install-run-n8n.md"
+
+## Step two: Open a workflow template
+
+n8n provides a quickstart template using training nodes. This allows you to work with fake data, and avoids setting up credentials.
+
+1. On the **Workflow templates** view, search for `Very quick quickstart`.
+2. Select the **Very quick quickstart** template to preview it.
+3. Select **Use this workflow** to open the template in your own editor.
+
+This is a basic workflow. It:
+
+1. Gets example data from the Customer Datastore node.
+2. Uses the Set node to extract just the data you want, and assign that data to variables. In this example, you use the customer name, ID, and description.
+
+Double click a node to explore its settings and how it processes data.
+
+## Step three: Run the workflow
+
+Run the workflow to check it's working, and load in data for the next steps: select **Execute workflow**.
+
+## Step four: Add a node
+
+Add a third node to message each customer and tell them their description. TheCustomer Messenger node allows you to send a message to fake recipients.
+
+1. Select the **Add node** <span class="inline-image">![Add node icon](/_images/try-it-out/add-node-small.png)</span> connector on the Set node.
+2. Search for **Customer Messenger**. n8n shows a list of nodes that match the search.
+3. Select **Customer Messenger (n8n training)** to add the node to the canvas. n8n opens the node automatically.
+4. You're going to use [expressions](/code-examples/expressions/) to map in the **Customer ID** and create the **Message**:
+    1. Next to **Customer ID**, select **Parameter options** <span class="inline-image">![Parameter options icon](/_images/try-it-out/parameter-options.png)</span> > **Add Expression**. n8n opens the expressions editor for this field.
```

Comment:
```suggestion
    1. Next to the **Customer ID** field, select **Parameter options** <span class="inline-image">![Parameter options icon](/_images/try-it-out/parameter-options.png)</span> > **Add Expression**. n8n opens the expressions editor for this field.
```

Without mentioning the field, I tend to look for information right next to the text.
Also, the first icon next to the field is a warning, not the gear for parameter options (but below next to the Message field they're in reverse order – confusing).

Reply:
I'm going to leave this one as is. It's a common convention to just use the UI name, rather than saying "the field" / "the button" etc. constantly (keeps the writing simpler and cleaner)